README.md

```diff
@@ -42,7 +49,24 @@ The following requirements should be met:
 * The code should be production grade. 
 * It should compile and run.
 
-### Tips/Advise
+### 2. Already written project
+
+We would be happy if you would submit a project you already have (e.g. potentially a demo project for another company). Still  the project has to obey to the following requirements:
+
+* Use Swift 3.0.
+* It should have at least two distinct network calls.
+* It should parse the network response and present the information to the user.
+* It should have some sort of persistence mechanism.
+* It should compile and run.
+* **Ideally** it should have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish). 
+
+If you have a project with these requirements then perfect! Please ensure you have a description of what the project does, in order to give us some context. ❤️
+
+### 3. Open Source work
+
+We would like to see a **non-trivial** pull request you have done to a public open source project. This should be something you are proud and where you show your technical skills. **It should also be related to iOS development**: it should be aligned with what you will do on a day-to-day basis. 😊✨🌳
```

Comment:
It is clear reading the entire text, sometimes it can be dangerous to look at the diff only.